data/pubmed-map.tsv

```diff
@@ -569,6 +573,8 @@ journal_nlm_id	scopus_id
 0154572	27371
 0154627	60402
 0154720	17025
+0154720	99256
```

Comment:
Note that the mapping is not bijective here. Looking into this:

- https://www.ncbi.nlm.nih.gov/nlmcatalog/0154720 is "Pflügers Archiv : European journal of physiology"
- https://scopus.com/sourceid/17025 is "Pflugers Archiv European Journal of Physiology"
- https://scopus.com/sourceid/99256 is XXXX (will replace when scopus webpage loads)

Reply:
Sorry it wasn't more helpful. Still, this gives a way to spot issues like that.